backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -223,6 +223,11 @@ def get_ui_display(self, data):
             ui_display = json.loads(get_value(data, "metadata._ui_display", ""))
             if is_superuser_or_cataloger_logged_in():
                 ui_display["can_edit"] = True
+                search_type = request.values.get("search_type", "", type=str)
+                if search_type == "hep-author-publication":
```

Comment:
do we really need the `search_type`, content type is not enough to distinguish the request we could then have the serialize just for this `content-type` without checking in both pleases for `search_type == "hep-author-publication"`?

Reply:
so you are saying to use a different ```content-type``` for the lit search request for author publications? I don't think a new ```content-type``` is needed, I think it is enough to use ```search_type``` in this case